api/proto/info/info.proto

```diff
@@ -347,6 +347,9 @@ message ZInfoDevice {
 
   // Information about hardware capabilities
   Capabilities capabilities = 42;
+
+  // Is there a profile override from a local_profile_server?
```

Comment:
We typically don't echo the config received by the controller in the info. We have to indicate the override since it didn't come from the controller. That's why I didn't put current+origin as you suggest.

Reply:
So, it will be non-empty in case of override, and empty in other cases? Can local_manager set empty profile?

Reply:
Ah, well, it can set empty, but it will not affect the behavior, got it.